src/midi/fluid_midi.c

```diff
@@ -1660,6 +1664,11 @@ new_fluid_player(fluid_synth_t *synth)
         player->track[i] = NULL;
     }
 
+    for(i = 0; i < synth->midi_channels; i++)
+    {
+        player->channel_isplaying[i] = 0;
```

Comment:
You probably want to do this array-initialization in fluid_player_reset(). Otherwise this array will never be updated between different files.

Reply:
And for readability you could use `FALSE` instead of `0`. Same for `1` above.

Reply:
done